docs/howto/helmdeploy.md

```diff
@@ -100,7 +94,7 @@ The following table lists the configurable parameters of the azure-service-opera
 | `azureClientSecret`  | Azure Service Principal Client Secret | `` |
```

Comment:
This is also documented in the charts readme -- actually it looks like this file and that readme are almost the same?
Can we avoid documentation duplication (which will ideally help us keep the right things up to date) somehow here?

Reply:
@frodopwns might know more about why there are two of these and if one can win

Reply:
I vote we get rid of the README in the charts directory.